tools/mapmerge2/requirements.txt

```diff
@@ -1,3 +1,3 @@
-pygit2==0.27.2
+pygit2==1.0.1
 bidict==0.13.1
 Pillow==6.2.0
```

Comment:
`pygit2` works, but could you also change `Pillow==6.2.0` to `Pillow==7.1.1`, as that works on the latest Python (3.8.2)